ext/filter/filter.stub.php

```diff
@@ -2,16 +2,16 @@
 
 /** @generate-function-entries */
 
-function filter_has_var(int $type, string $variable_name): bool {}
+function filter_has_var(int $type, string $var_name): bool {}
 
-function filter_input(int $type, string $variable_name, int $filter = FILTER_DEFAULT, array|int $options = 0): mixed {}
+function filter_input(int $type, string $var_name, int $filter = FILTER_DEFAULT, array|int $options = 0): mixed {}
 
-function filter_var(mixed $variable, int $filter = FILTER_DEFAULT, array|int $options = 0): mixed {}
+function filter_var(mixed $var, int $filter = FILTER_DEFAULT, array|int $options = 0): mixed {}
```

Comment:
```suggestion
function filter_var(mixed $value, int $filter = FILTER_DEFAULT, array|int $options = 0): mixed {}
```
As far as I know, we only use `$var` for by-reference parameters that actually require a variable. Otherwise we use `$value`.